components/concept/toolbar/build.gradle

```diff
@@ -19,6 +19,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    buildFeatures {
+        viewBinding true
+    }
```

Comment:
You added this to some components, but they don't seem to use it? 🤔

Reply:
I was under the impression at the time that similar to data binding one must enable binding for all modules that have dependencies on modules that use data binding, even if the module doesn't directly use data binding. 
Should I go through them and remove them from the ones that do not use view binding?

Reply:
Oh, interesting. If it's required then I guess we need to, but if we can remove them that would be nice :)